mappings/net/minecraft/inventory/ItemSlot.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_5630 net/minecraft/inventory/ItemSlot
```

Comment:
This might be confused with the other item slot class, `Slot` which is used by menus/SHs.

Reply:
So `EntityItemSlot`? `ArgumentItemSlot`?

Reply:
I will use `CommandItemSlot` as this represents an item slot for command access.